src/components/BaseTable/BaseTable.tsx

```diff
@@ -116,6 +118,66 @@ export const BaseTable = React.forwardRef(
         const footerRowCount = footerGroups ? footerGroups.length : 0;
         const rowCount = bodyRowCount + headerRowCount + footerRowCount;
 
+        const renderBodyRows = React.useCallback(() => {
+            const bodyRows = rowVirtualizer?.getVirtualItems() || rows;
+
+            if (bodyRows.length === 0) {
+                return (
+                    <tr>
```

Comment:
Use `<BaseRow>` (it has to be modified in advance) or add some of row attributes here (className at least)